cypress/fixtures/docs/staticJSONObjects.md

```diff
@@ -0,0 +1,144 @@
+# Static JSON Objects
+
+Firebolt Calls has two types of JSON objects: Firebolt Calls and Validation Objects. These objects are used to make API calls and validate the response received from the API.
+
+## [Supported Static Glue Codes](../../support/step_definitions/fireboltCalls.md)
+
+## Firebolt Calls
+
+Firebolt object is used to make an API call. It contains the following parameters:
+
+- method - Name of the API to make an API call
+- params - Represent the parameters to be sent for a Firebolt call
+- context - Represent the data that needs to be stored in the API/event object that helps to search for a specific object which contains the response in a global list
+- expected - Determine whether expecting for an error or result
+
+### How to Configure Firebolt Calls
```

Comment:
From line 19, you can remove **'params, context, and expected are optional parameters. By default, they are set to below values:' and lines 20/21/22 .** 
Those are already added in above table

Reply:
done